src/main/java/org/jabref/gui/entryeditor/OtherFieldsTab.java

```diff
@@ -0,0 +1,36 @@
+package org.jabref.gui.entryeditor;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javafx.scene.control.Tooltip;
+
+import org.jabref.Globals;
+import org.jabref.gui.BasePanel;
+import org.jabref.gui.IconTheme;
+import org.jabref.gui.JabRefFrame;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.EntryType;
+
+public class OtherFieldsTab extends FieldsEditorTab {
+    public OtherFieldsTab(JabRefFrame frame, BasePanel basePanel, EntryType entryType, EntryEditor parent, BibEntry entry) {
```

Comment:
Very minor, but please add a blank between the class declaration and the method.